assets/js/components/search.js

```diff
@@ -1,22 +1,57 @@
 'use strict';
 
 const $ = global.jQuery;
+const Mark = require('mark.js');
 
 class Search {
 
     constructor(el, trees) {
         this._el = $(el);
         this._trees = trees;
         this._input = this._el.find('[data-role="input"]');
+        this._clearButton = this._el.find('[data-behavior="clear-search"]');
+
+        this._marker = new Mark('.Tree-items');
```

Comment:
can we init this with `this._trees`? then it would be scoped to the class instance, and not touch areas outside.

Reply:
Done